with the improvement it looks like this:
```

~/go/src/github.com/juju/juju/caasoperator⟫ make test_env_var JUJU_K8S_MODEL=k8smod
ubuntu@ip-172-31-21-187:~/go/src/github.com/juju/juju/caasoperator⟫ make test_env_var
Makefile:43: *** Undefined JUJU_K8S_MODEL.  Stop.

~/go/src/github.com/juju/juju/caasoperator⟫ make test_env_var JUJU_K8S_MODEL=k8smodf
ERROR getting model details: model k8scon:admin/k8smodf not found
Makefile:43: recipe for target 'test_env_var' failed
make: *** [test_env_var] Error 1

~/go/src/github.com/juju/juju/caasoperator⟫ make local-update JUJU_K8S_MODEL=k8smod
make -C/home/ubuntu/go/src/github.com/juju/juju install
make[1]: Entering directory '/home/ubuntu/go/src/github.com/juju/juju'
skipping godeps
...
```

--- a/caasoperator/Makefile
+++ b/caasoperator/Makefile
@@ -39,11 +39,11 @@ add-dockerhub-secret:
 
 local-update-restart: local-update restart-operators
 
-test_env_var:
+check-k8s-model:
 	@:$(if $(value JUJU_K8S_MODEL),, $(error Undefined JUJU_K8S_MODEL))
 	@juju show-model ${JUJU_K8S_MODEL} > /dev/null
 
-local-update: test_env_var image
+local-update: check-k8s-model image
 	$(eval kubeworkers != juju status -m ${JUJU_K8S_MODEL} kubernetes-worker --format json | jq -c '.machines | keys' | tr  -c '[:digit:]' ' ' 2>&1)
 	docker save ${DOCKER_USERNAME}/caasoperator | gzip > /tmp/caasoperator-image.tar.gz
 	$(foreach wm,$(kubeworkers), juju scp -m ${JUJU_K8S_MODEL} /tmp/caasoperator-image.tar.gz $(wm):/tmp/caasoperator-image.tar.gz ; )

